src/server/rpc/procedures/hawaii-co2-data/hawaii-co2-data.js

```diff
@@ -0,0 +1,103 @@
+const lineReader = require('line-reader');
```

Comment:
It's a pretty minor thing but I am not sure if this merits adding another dependency for reading the file in (especially since the file only has 800 lines or so). This would be fine to either use `fs.readFileSync` or `fs.readFile` instead. It also wouldn't be much more code. Something like the following would suffice:
```
const fs = require('fs');
fs.readFileSync(path.join(__dirname,'co2_mm_mlo.txt'), 'utf8').split('\n')
   .forEach(function(line) {
        // Add your function passed to `lineReader.eachLine` here
    });
```